src/System.Collections.Specialized/src/System/Collections/Specialized/OrderedDictionary.cs

```diff
@@ -270,22 +285,31 @@ public OrderedDictionary AsReadOnly()
         /// </devdoc>
         public bool Contains(object key)
         {
-            return objectsTable.Contains(key);
+            if (_objectsTable == null)
```

Comment:
This is causing this test to fail
```
        public void ContainsTests()
        {
            var d = new OrderedDictionary();
            Assert.Throws<ArgumentNullException>(() => d.Contains(null));
```
because you are returning false, when previously it would look for null in an empty table, which throws.

I don't feel strongly about retaining the old behavior (and update test), but I think I'd be inclined to just keep the behavior and not break anyone by changing to `if (_objectsTable == null && key != null)`

Reply:
I added
```c#
            if (key == null)
            {
                throw new ArgumentNullException(nameof(key));
            }
```
 we use the same in Remove method.